### ISSUE TITLE
Build warning: A  implementation of Method 'addJavaLttngImport' ...

### ISSUE THREAD

**Issue by Nava2** [umple/umple#664]
When building the current repository: 

```
A  implementation of Method 'addJavaLttngImport' in class 'JavaGenerator' is already defined at line 1575 of 'Generator_CodeJava.ump' file. New definition at line 1592 of 'Generator_CodeJava.ump' file is being disconsidered
```

Side note: Disconsidered should be replaced with disregarded.

**Comment by TimLethbridge** [comment]
The error message had been fixed some time ago. I am going to do a PR that cleans up the error message.

**Comment by TimLethbridge** [comment]
Fixed in #1196
